### PR TITLE
Add edes csv to library and ceqr-survey

### PR DESCRIPTION
**Pull request by fvankrieken**
Continuing #575

### CODE REVIEW

File: products/ceqr_survey/models/intermediate/int__edesignation_flags.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+SELECT
+    bbl,
+    'edesignation_air' AS variable,
+    enumber AS id
+FROM dcp_edesignation
+WHERE air_code = '1'
+UNION ALL
+SELECT
+    bbl,
+    'edesignation_hazmat' AS variable,
```
[the remark this answers is not in the document]
Comment (fvankrieken) [reply products/ceqr_survey/models/intermediate/int__edesignation_flags.sql:21]: Nope!

File: products/ceqr_survey/build_metadata.json
Comment (damonmcc) [inline products/ceqr_survey/build_metadata.json:1]: ✂️?

**Comment by fvankrieken** [comment]
Let me actually add table definitions as well

**Comment by fvankrieken** [comment]
Ready for review once more. Tests for these tables pass

**Review by damonmcc — APPROVED** [review]

**Pull request merged**